lib/suites/Ed25519Signature2018.js

```diff
@@ -40,7 +40,21 @@ module.exports = class Ed25519Signature2018 extends LinkedDataSignature {
       JSON.stringify(header), {forge});
 
     let encodedSignature;
-    if(this.injector.env.nodejs) {
+    const {nodejs} = this.injector.env;
+    if(options.signer) {
+      // a custom signing API has been provided
+      // a Uint8Array is passed to the signer
+      const {data} = _createSigningString({nodejs});
+      let bytesUint8Array;
+      if(nodejs) {
+        // convert Buffer to Uint8Array
+        bytesUint8Array = new Uint8Array(data.buffer);
+      } else {
+        // convert binary encoded string to Uint8Array
+        bytesUint8Array = forge.util.binary.raw.decode(data);
+      }
+      encodedSignature = await options.signer.sign({bytesUint8Array});
```

Comment:
Recommend we name it `data` or `message` instead of `bytesUint8Array` as a more common sign API parameter name.

Reply:
using `message`. Done.